tests/index.mocha.js

```diff
@@ -831,7 +831,7 @@ describe('Providing bad glyphs', () => {
       unicode: ['\uE002'],
     };
 
-    let firstError = true;
+    /*let firstError = true;
```

Comment:
If the code is not useful anymore, best is to remove it then comment it. Why is it removed btw?

Reply:
Because when I change the sax library for saxasync, the latter does not throw the error that the original one does and I don't know why or if that test is really necessary.